src/sidebar/util/test/build-thread-test.js

```diff
@@ -225,17 +231,24 @@ describe('sidebar/util/build-thread', function () {
       ]);
     });
 
-    it('threads new annotations which have tags but not IDs', function () {
+    it('threads and sorts new annotations which have tags but not IDs', () => {
       const fixture = [
         {
           $tag: 't1',
         },
+        { $tag: 't5' },
+        { $tag: 't2' },
+        { $tag: 't9', references: ['foo'] },
       ];
       const thread = createThread(fixture);
-      assert.deepEqual(thread, [{ annotation: fixture[0], children: [] }]);
+      assert.deepEqual(thread[0], { annotation: fixture[0], children: [] });
+      // Default sort is by tag, so `t2` before `t5`
+      assert.deepEqual(thread[1], { annotation: fixture[2], children: [] });
```

Comment:
Ah, I see we are testing the default sorting. If we don't have a use for in the app, we could still consider removing it.

Reply:
It has been removed.